Prace_domowe/Praca_domowa4/Tomasz_Krupinski/PD4.ipynb

```diff
@@ -0,0 +1,1219 @@
+{
```

Comment:
<p>To chyba nawet nie jest kwestia jupytera tylko pythona, scikit learn niestety nie obsługuje domyślnie GPU. Znalazłem taką bibliotekę: https://mklab.iti.gr/results/gpu-accelerated-libsvm/ jednak ona obsługuje tylko jeden kernel. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>